src/ocamlorg_frontend/pages/blog.eml

```diff
@@ -73,15 +70,14 @@ Layout.render
             <div class="lg:pl-16 space-y-12">
                 <h2 class="text-lg mt-0 font-semibold border-b border-current text-primary-600 pb-3">News</h2>
 
-                <% news |> List.iter (fun (item : Data.News.t) -> %>
+                <% news |> List.iter (fun (item : Data.Planet.Post.t) -> %>
```

Comment:
The solution here isn't to change `Data.News.t` to `Data.Planet.Post.t`, but instead to not try to render authors for the news posts.

Reply:
Hello Sabine, thank you for your response. It’s not a lie if I tell you that I am confused at this point however, I’d appreciate more clarity and your assistance on this task. Thank you so much.

Reply:
no, you aren't supposed to createa an authors field on that type